pkg/skaffold/docker/image.go

```diff
@@ -133,10 +132,7 @@ func addTag(ref name.Reference, targetRef name.Reference, auth authn.Authenticat
 		return err
 	}
 
-	wo := remote.WriteOptions{
-		MountPaths: []name.Repository{ref.Context()},
-	}
-	return remote.Write(targetRef, img, auth, t, wo)
+	return remote.Write(targetRef, img, auth, t, remote.WriteOptions{})
```

Comment:
I think we want to keep MountOptions if we can.

Reply:
MountPaths field was removed. I don't know if something else needs to be done. @jonjohnsonjr @mattmoor ?

Reply:
The happy path for mounting is now handled by the library, see https://github.com/google/go-containerregistry/pull/186

tl;dr: remote layer reads retain the source repo and we specialize that case on remote layer writes.

Reply:
Ah, nice. Missed that change in the library somehow.